plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -52,20 +52,20 @@
 	 * in name or tags.
 	 * 
 	 * @param filterText
-	 * @param template
+	 * @param resource
 	 * @return
 	 */
-	public static boolean isMatching(final String filterText, IResource template) {
+	public static boolean isMatching(final String filterText, IResource resource) {
```

Comment:
I guess that a guard for resource == null (like the one we already have for the filterText or maybe even assert vua org.eclipse.core.runtime.Assert that throws an assertion (runtime-) exception?) would be a great idea?